.github/workflows/postgresql-versions.yml

```diff
@@ -0,0 +1,145 @@
+name: PostgreSQL tests
+
+on:
+  workflow_dispatch: # allow manual runs
+  schedule:
+    - cron:  '0 0 * * *' # Runs every day at midnight
+
+jobs:
+  build:
+    name: "Experimental PostgreSQL ${{ matrix.postgresql-versions }} Test (PHP: ${{ matrix.php-versions }}, deps: ${{ matrix.dependencies }})"
+
+    continue-on-error: true
+
+    strategy:
+      fail-fast: false
+      matrix:
+        flow-versions: ['master']
+        php-versions: ['8.1', '8.0']
+        # See https://www.postgresql.org/support/versioning/
+        # Use https://hub.docker.com/_/postgres for available versions
+        postgresql-versions: ['14-alpine', '13-alpine', '12-alpine', '11-alpine', '10-alpine']
+        dependencies: ['highest']
+        composer-arguments: ['--ignore-platform-reqs'] # to run --ignore-platform-reqs in experimental builds
```

Comment:
I think the `--ignore-platform-reqs` isn't required here. It's to allow installing composer dependencies that are prohibited for the given PHP version, so mostly useful for nightly/latest tests